build.sh

```diff
@@ -42,18 +42,18 @@ case "$(uname -s)" in # adjust compilation option based on platform
     Linux)
         echo 'Compiling for Linux…'
         sys_cflags='-march=native -Wno-error=redundant-decls -Wno-error=unused-but-set-variable'
-        sys_opts='--enable-kvm --disable-xen --disable-werror'
+        sys_opts='--enable-kvm --disable-xen --disable-werror --audio-drv-list=alsa'
         ;;
     Darwin)
         echo 'Compiling for MacOS…'
-        sys_cflags='-march=native'
+        sys_cflags='-march=native --audio-drv-list=coreaudio'
```

Comment:
`--audio-drv-list=coreaudio` needs to go into `sys_opts` instead of `sys_cflags` (same for Windows).

Reply:
Duh, thx